src/main/webapp/app/shared/statistics-graph/statistics-score-distribution-graph.component.ts

```diff
@@ -0,0 +1,135 @@
+import { Component, Input, OnInit, ViewChild } from '@angular/core';
+import { StatisticsService } from 'app/shared/statistics-graph/statistics.service';
+import { ChartDataSets, ChartOptions, ChartType } from 'chart.js';
+import { BaseChartDirective, Label } from 'ng2-charts';
+import { DataSet } from 'app/exercises/quiz/manage/statistics/quiz-statistic/quiz-statistic.component';
+import { TranslateService } from '@ngx-translate/core';
+import { GraphColors, SpanType } from 'app/entities/statistics.model';
+import { round } from 'app/shared/util/utils';
+
+@Component({
+    selector: 'jhi-statistics-score-distribution-graph',
+    templateUrl: './statistics-score-distribution-graph.component.html',
+})
+export class StatisticsScoreDistributionGraphComponent implements OnInit {
+    @Input()
+    averageScoreOfExercise: number | undefined;
+    @Input()
+    scoreDistribution: number[] | undefined;
+    @Input()
+    numberOfExerciseScores: number | undefined;
+
+    // Html properties
+    LEFT = false;
+    RIGHT = true;
+    SpanType = SpanType;
```

Comment:
Where are these used? I cannot see them in the template

Reply:
True, I kept this variable which is not used anymore. Removed it